src/models/api/VariableV2Request.ts

```diff
@@ -0,0 +1,11 @@
+export type VariableV2CreateRequest = Partial<{
+  name: string | null,
+  value: unknown,
```

Comment:
Is this the only difference between v1 and v2? That previously this was a string, but now it could be... what - an object?

Reply:
any json compatible type!